site/write_ups/2022/16-11-22_writeup.md

```diff
@@ -0,0 +1,128 @@
+
+Interesting point in the article that the commercial success is not necersarily related to quality. There's clearly a difference between someone who just typed in some prompts vs. someone who takes hour over editing it and stuff like that. There's this feeling of "Quantity being consumed into quality" where we don't know what's going happen but it will change who makes art and what we call art. Everything has value to someone. 
+
+## Who should decide whether art is high-quality?
+
+The article says that the success of an artist should be based on the quality of their art rather than on its popularity with consumers. But who gets to make that call?
+
+Would the art critics have rated this as highly if they were as familiar with the style that this AI always generates? We think not. Art can cater to different people: is it impressive because of the process or because of the outcome? Does success in art mean financial gain? That's not necessarily a good measure for what art means.
+
+Is it cheaper to generate AI art? Ones we've seen look pretty expensive considering they're made digitally. But are we undervaluing the art created by AI artists? There is some additional curation involved in creating AI art. Knowing how something was made makes us appreciate something more. 
+
+How do we value aesthetics? Is some of it what people believe about their own art? There is definitely capitalism and social bias in art. If (rich) people want a piece then it's consider more valuable. 
+
+Something not mentioned in the article but seems like a big issue is to do with value of the artist themselves. Like with restaurants - you'll go to a famous chef's restaurant but the chef won't cook your meal. It's their ideas that you are paying for. 
+
+Should skill be compensated? If you train for years to hone skill to create a certain type of art. With AI tools, now you can just type in a sentence and get out an image with little to no extra input. Does the quality of art scale somehow with effort that went in to produce it or skill level required? Our gut instincts may say yes, but is that actually true?
+
+Will there become a problem with "Seen one seen them all" where AI generated art becomes too recognisable? Perhaps there is a comparison to fan fiction; it's usually non-comercial, so no ethical problems here. Most fan fiction isn't good; should the law be different if the art isn't good? Can we have a cultural shift around how you value things? Around how things sell?
+
+At the end of the day, art is what you make of it! On a purely philosophical level, it has value if it speaks to you, that’s it! Value of art is culturally constructed, Somali women creating structures by weaving don’t view it as art, but non-Somali cultures may well see it as art! It's highly contextual.	Does the desire define the value? Desire can be derived from various places, may be due to aesthetic value, or the exclusivity, or the authorship. 
```

Comment:
```suggestion
At the end of the day, art is what you make of it! On a purely philosophical level, it has value if it speaks to you, that’s it! The value of art is socially and culturally constructed.
For example, Somali women who create structures by weaving may not view it as art, but in non-Somali cultures it may be more commonly seen as art. It's highly contextual.	
Does the desire define the value? Desire can be derived from various places, may be due to aesthetic value, or the exclusivity, or the authorship. 
```